src/plone/recipe/zope2instance/__init__.py

```diff
@@ -114,7 +114,8 @@ def install(self, update=False):
             # does not return an iterable anymore with 'parsed_version',
             # instead a class-object is returned. Let's re-parse that, to
             # support setuptools > 37.8.0, while keeping backwards-compat:
-            if not isinstance(parsed, tuple):
+            if (not isinstance(parsed, tuple)
+                and hasattr(parsed, "base_version")):
```

Comment:
This should be an improvement, but I do wonder: if `parsed` is not a tuple **and** does not have a `base_version`, then what is it and can we safely call `parsed[0:2]` on it?
If needed, we could do a big try/except around everything, and set `major=2` and `minor=13` if it really fails.
But really, if Jenkins passes now, which I think it will, I am happy.

Reply:
Do you mean you'd rather see it fail if `parsed` does not have a `base_version` ?

Reply:
No, your code is fine. But since you tackle a possible corner case with your fix, I just wondered if there are other corner cases.

Reply:
I wondered how useful what I added was. I agree we are still not fully covered... But good to me as well